### ISSUE TITLE
Question with logit_scale on your code

### ISSUE THREAD

**Issue by huisu0103** [Fsoft-AIC/Open-Vocabulary-Affordance-Detection-in-3D-Point-Clouds#2]
Thank you for your github.
I appreciate your paper and code sharing on github!

I have a question with logit_scale on your code.
There is a logit_scale in 'models/openad_pn2.py' like this 'self.logit_scale = nn.Parameter(torch.ones([]) * np.log(1 / 0.07))' on class OpenAD_PN2.

I wonder how this self.logit_scale was determined. Can you explain it in detail?

Thank you for your Github!

**Comment by toannguyen1904** [comment]
Hi @HuisuIntwnr. Thank you very much for your interest in our work.

For the conciseness, we did not mention the ```logit_scale``` in our paper. In our implementation, the ```logit_scale``` is a learnable temperature parameter, which is updated during training. The purpose of this parameter is to control the range of the logits in the softmax. The use of the logit_scale is mentioned in the CLIP paper, which you can refer to [here](https://proceedings.mlr.press/v139/radford21a/radford21a.pdfcontrols%20the%20range%20of%20the%20logits%20in%20the%20softmax). If you wish to delve deeper into the importance of this temperature parameter, you can refer to [this paper](https://openaccess.thecvf.com/content_cvpr_2018/papers/Wu_Unsupervised_Feature_Learning_CVPR_2018_paper.pdf) as well.

Best regards,
Toan.
